contents/relayer.ts

```diff
@@ -1,41 +1,27 @@
 import type { PlasmoCSConfig } from "plasmo"
 
 import { sendToBackground } from "@plasmohq/messaging"
-import { relay } from "@plasmohq/messaging/relay"
-
-import { WaalletMessage } from "~packages/waallet/message"
 
 export const config: PlasmoCSConfig = {
   matches: ["<all_urls>"],
   run_at: "document_start"
 }
 
-relay(
-  {
-    name: "mCreateWindow"
-  },
-  async (req) => {
-    console.log(`[content][relay][mCreateWindow][request]`, req)
-    const res = await sendToBackground(req as any)
-    console.log(`[content][relay][mCreateWindow][response]`, res)
-    return res
-  }
-)
+const channelToMainWorld = new BroadcastChannel("MainWorld")
+const channelToContentScript = new BroadcastChannel("ContentScript")
 
-relay(
```

Comment:
Since `@plasmohq/messaging/relay` in `contents/relayer.ts` is removed, consider removing `packages/messenger/background/relay.ts` file.